example/fittemplate.md

```diff
@@ -0,0 +1,137 @@
+
+## Loading the data
+
+Before starting with FieldTrip, it is important that you set up your
+[MATLAB path](/faq/should_i_add_fieldtrip_with_all_subdirectories_to_my_matlab_path) properly.
+
+You can load the head shape measured during the MEG recording with the Polhemus and a
+template volume conduction model. We have to ensure that they have consistent units, hence 
+we will convert the units into mm.
+
+    polhemus = ft_read_headshape(filename);
+    polhemus = ft_convert_units(polhemus,'mm');
+
+    template = ft_read_vol('standard_bem.mat');
+    template = ft_convert_units(template,'mm');
+
+## Coregistration
+
+In the next step we coregister the template head model with the Polhemus head shape to ensure that they are expressed consistently in the MEG coordinate system (i.e. relative to the same origin and with the axes pointing in the same direction). The coregistration consists of a rotation and translation.
+
+    cfg = [];
+    cfg.template.headshape      = polhemus;
+    cfg.checksize               = inf;
+    cfg.individual.headmodel    = template;
+    cfg                         = ft_interactiverealign(cfg);
+    template                    = ft_transform_geometry(cfg.m, template);
+
+## Determining and applying of the transformation
+
+### Method 1: On the basis of fiducials
+
+For the first approach we will fit a sphere to the three fiducials and we will fit another sphere to the template head model. On the basis of difference of the center and scaling between the two spheres we can derive a transformation containing information of the translation and the global scaling.
+
+For this approach we will need determine the fiducials to Fiducials
+
+    nas = polhemus.elecpos
+    lpa = polhemus.elecpos
+    rpa = polhemus.elecpos
+
+Fit a sphere to the MRI template
+
+    cfg=[];
+    cfg.method='singlesphere';
+    scalp_sphere = ft_prepare_headmodel(cfg, template.bnd(1));
+
+Fit a sphere to the fiducials
+
+    cfg=[];
+    cfg.method = 'singlesphere';
+    headshape_sphere = ft_prepare_headmodel(cfg, fiducials);
+
+Determine the global scaling and translation
+
+    scale = headshape_sphere.r/scalp_sphere.r;
+
+    T2 = [1 0 0 scalp_sphere.o(1);
+          0 1 0 scalp_sphere.o(2);
+          0 0 1 scalp_sphere.o(3);
+          0 0 0 1                ];
+
+    T1 = [1 0 0 -scalp_sphere.o(1);
+          0 1 0 -scalp_sphere.o(2);
+          0 0 1 -scalp_sphere.o(3);
+          0 0 0 1                 ];
+
+    S  = [scale 0 0 0;
+          0 scale 0 0;
+          0 0 scale 0;
+          0 0 0 1 ];
+
+    transformation = T1*S*T2;
+
+Appyly the transformation to the template head model
+
+    template_fiducial = ft_transform_geometry(transformation, template)
+
+Now that we have have the surfaces of the full template model (not only the scalp) rotated, translated and scaled to fit the Polhemus measurement, we can recompute the volume conduction model 
+
+    cfg = [];
+    cfg.method = 'bemcp';
```

Comment:
Why would you use bemcp here? It does not support MEG forward models, which is the prime use case. Using singleshell makes more sense, or openmeeg.

Reply:
I changed it to openmeeg